packages/tables/resources/views/components/pagination/item.blade.php

```diff
@@ -17,9 +17,9 @@
             'focus:text-primary-600' => (! $active) && (! $disabled) && (! $icon) && (! $separator),
             'transition' => ((! $active) && (! $disabled) && (! $separator)) || $active,
             'text-primary-600' => ((! $active) && (! $disabled) && $icon && (! $separator)) || $active,
-            'focus:underline bg-primary-500/10 ring-2 ring-primary-500' => $active,
-            'cursor-not-allowed pointer-events-none opacity-70' => $disabled,
-            'cursor-default' => $separator,
+            'focus:underline bg-primary-500/10 ring-2 ring-primary-500 filament-tables-pagination-item-active' => $active,
+            'cursor-not-allowed pointer-events-none opacity-70 filament-tables-pagination-item-disabled' => $disabled,
+            'cursor-default filament-tables-pagination-item-seperator' => $separator,
```

Comment:
Could you please move these custom classes to the start of the class strings?

Reply:
done